locksmith/src/operations/userOperations.ts

```diff
@@ -195,14 +214,55 @@ export const eject = async (publicKey: ethereumAddress): Promise<any> => {
 }
 
 export const findByEmail = async (emailAddress: string) => {
-  const user = await UserReference.findOne({
+  const user = await UserAccount.findOne({
```

Comment:
`UserAccount` and `UserReference` are different types. Can you ensure that the app still works here when the returned values are different?

Reply:
The function `findByEmail` returns different types of objects... which I think is a bit risky and something you need to add tests for and make sure the rest of the application can handle.